### PR TITLE
Pagetree rewrite

### PR DESCRIPTION
**Pull request by grebaldi**
Though not a full rewrite, this PR introduces still some changes:

- Retrieve nodes directry from the CR part of the store, without further conversion
- More flat prop structure for the tree component
- Add drag and drop functionality

### CODE REVIEW

File: packages/neos-ui/src/Containers/LeftSideBar/PageTree/Node/index.js [hunk 1]
```diff
@@ -48,68 +60,141 @@ export default class Node extends PureComponent {
...
+        const {node, onNodeDrag} = this.props;
+
+        onNodeDrag(node);
+    }
+
+    handleNodeDrop = () => {
+        const {node, onNodeDrop} = this.props;
+
+        onNodeDrop(node);
+    }
```
Comment (skurfuerst) [inline packages/neos-ui/src/Containers/LeftSideBar/PageTree/Node/index.js]: why is the function above written differently? :)

Comment (grebaldi) [inline packages/neos-ui/src/Containers/LeftSideBar/PageTree/Node/index.js]: Basically to save one `.bind(this)` ;)

Comment (skurfuerst) [inline packages/neos-ui/src/Containers/LeftSideBar/PageTree/Node/index.js]: I see :)

**Comment by skurfuerst** [comment]
very nice, I'm merging this now <3

**Pull request merged**